<launch>

<node pkg="fake_odometry" type="odom_publisher_node" name="odom_publisher_node" output="screen">
<remap from="/pose_update" to="/amcl_pose"/>
</node>

<node pkg="scan_from_image" type="laser_scan_from_image" name="laser_scan_from_image" output="screen">
<rosparam file="$(find scan_from_image)/launch/map_2/map.yaml" command="load" ns="/map_data"/>
<rosparam file="$(find scan_from_image)/launch/scan_params.yaml" command="load" ns="/scan_params"/>
<remap from="/pose_stamped" to="/pose_from_odom"/>
</node>
<node name="map_server" pkg="map_server" type ="map_server" args="$(find scan_from_image)/launch/map_2/map.yaml"/>
<node pkg="amcl" type="amcl" name="amcl" output="screen">
  <!-- Publish scans from best pose at a max of 10 Hz -->
  <remap from="scan" to="/scan"/>
  <param name="odom_model_type" value="diff-corrected"/>
  <param name="gui_publish_rate" value="-1.0"/>
  <param name="laser_max_beams" value="120"/>
  <param name="min_particles" value="20"/>
  <param name="max_particles" value="1000"/>
  <param name="kld_err" value="0.01"/>
  <param name="kld_z" value="0.99"/>
  <param name="initial_pose_x" value="24.50"/>
  <param name="initial_pose_y" value="10.080"/>
  <param name="initial_pose_a" value="0.0"/>
</node>
<node pkg="tf2_ros" type="static_transform_publisher" name="base_link_to_laser" args="0.0 0.0 0.0 0.0 0.0 0.0 /base_link /laser" />

</launch>

